python/pyspark/pandas/frame.py

```diff
@@ -4815,6 +4815,13 @@ def to_spark_io(
         index_col: Optional[Union[str, List[str]]] = None,
         **options
     ) -> None:
+        """An alias for :func:`spark.to_spark_io`.
+        See :meth:`pyspark.pandas.DataFrame.spark.to_spark_io`.
+
+        .. deprecated:: 3.2.0
+            Use :func:`DataFrame.spark.to_spark_io` instead.
```

Comment:
What's diff here? `` :meth:`pyspark.pandas.DataFrame.spark.to_spark_io` `` vs ``  :func:`DataFrame.spark.to_spark_io` ``?

Reply:
Updated to `pyspark.pandas.spark.accessors.SparkFrameMethods.to_spark_io`. Please correct me if I'm wrong.